raiden/tests/unit/test_matrix_transport.py

```diff
@@ -167,7 +167,7 @@ def random_or_none(url):  # pylint: disable=unused-argument
 
     server_count = 9
     sorted_servers = sort_servers_closest([f"https://server{i}.xyz" for i in range(server_count)])
-    rtts = [rtt for (_, rtt) in sorted_servers]
+    rtts = [rtt for (_, rtt) in sorted_servers.items()]
```

Comment:
why no just iterate over `sorted_servers.values()`?

Reply:
Err, yeah. It was obviously too late already yesterday...